### PR TITLE
Add custom comparision operators

### PR DESCRIPTION
**Pull request by MaxGraey**
Some of this operators need for string comparison and additional sorting tests

- [x] operator "!="
- [x] operator ">"
- [x] operator ">="
- [x] operator "<"
- [x] operator "<="

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1528,6 +1528,7 @@
         "anymatch": "2.0.0",
         "async-each": "1.0.1",
         "braces": "2.3.1",
+        "fsevents": "1.1.3",
```
Comment (dcodeIO) [inline package-lock.json]: Any idea where this comes from? Maybe optional, and only used on Mac?

Comment (MaxGraey) [reply package-lock.json]: Yes, It's optional module only for mac which speedup file watching. It seems tslint want it

Comment (MaxGraey) [reply package-lock.json]: I can exclude this change from package-lock

File: package-lock.json [hunk 2]
```diff
@@ -4429,6 +4429,13 @@
       "integrity": "sha1-MHXOk7whuPq0PhvE2n6BFe0ee6s=",
       "dev": true
     },
+    "nan": {
```
Comment (dcodeIO) [inline package-lock.json]: Seems there's more :)

Comment (MaxGraey) [reply package-lock.json]: Done!

**Comment by dcodeIO** [comment]
Thanks! :)

**Pull request merged**